### PR TITLE
Replace a Few IntFlags with Uint64Flags

### PR DESCRIPTION
**Pull request by rauljordan**
As part of #9958, we should be only using integer types where we truly need signed numbers. This PR refactors a few low hanging fruit to use Uint64Flag instead of IntFlag in the beacon node configuration options, cc @uncdr

### CODE REVIEW

File: beacon-chain/p2p/testing/fuzz_p2p.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func (p *FakeP2P) DiscoveryAddresses() ([]multiaddr.Multiaddr, error) {
 }
 
 // FindPeersWithSubnet mocks the p2p func.
-func (p *FakeP2P) FindPeersWithSubnet(_ context.Context, _ string, _ uint64, _ int) (bool, error) {
+func (p *FakeP2P) FindPeersWithSubnet(ctx context.Context, topic string, subIndex uint64, threshold uint64) (bool, error) {
```
Comment (rkapka) [inline beacon-chain/p2p/testing/fuzz_p2p.go:64]: Why the change? Deepsource will complain about this.

File: beacon-chain/p2p/testing/mock_peermanager.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ func (m MockPeerManager) DiscoveryAddresses() ([]multiaddr.Multiaddr, error) {
 func (m MockPeerManager) RefreshENR() {}
 
 // FindPeersWithSubnet .
-func (m MockPeerManager) FindPeersWithSubnet(_ context.Context, _ string, _ uint64, _ int) (bool, error) {
+func (m MockPeerManager) FindPeersWithSubnet(ctx context.Context, topic string, subIndex uint64, threshold uint64) (bool, error) {
```
Comment (rkapka) [inline beacon-chain/p2p/testing/mock_peermanager.go:54]: Same here.

File: beacon-chain/p2p/testing/p2p.go [hunk 3]
```diff
@@ -349,7 +349,7 @@ func (p *TestP2P) Peers() *peers.Status {
 }
 
 // FindPeersWithSubnet mocks the p2p func.
-func (p *TestP2P) FindPeersWithSubnet(_ context.Context, _ string, _ uint64, _ int) (bool, error) {
+func (p *TestP2P) FindPeersWithSubnet(ctx context.Context, topic string, subIndex uint64, threshold uint64) (bool, error) {
```
Comment (rkapka) [inline beacon-chain/p2p/testing/p2p.go:352]: Same here.

**Review by rkapka — CHANGES_REQUESTED** [review]

File: beacon-chain/p2p/testing/fuzz_p2p.go (hunk 1 above)
Comment (rauljordan) [reply beacon-chain/p2p/testing/fuzz_p2p.go:64]: Goland refactor func signature did this :(

**Comment by rauljordan** [comment]
Ready @rkapka

**Review by prestonvanloon — APPROVED** [review]

**Review by rkapka — APPROVED** [review]
